helpers/database/discount.php

```diff
@@ -12,15 +11,13 @@ function getNewID($databaseConnection) {
 
 function getCurrentDiscounts($databaseConnection)
 {
-    removeExpiredDiscounts($databaseConnection);
     $query = "SELECT SpecialDealID, StockItemID, StockItemName, DiscountPercentage, StartDate, EndDate FROM specialdeals JOIN stockitems USING (StockItemID)";
     $result = mysqli_query($databaseConnection, $query);
     return mysqli_fetch_all($result, MYSQLI_ASSOC);
 }
 
 function productHasDiscount($id, $databaseConnection)
 {
-    removeExpiredDiscounts($databaseConnection);
```

Comment:
Je haalt nu de removeExpiredDiscount weg maar ik zie nog geen check of de actie al verlopen is loopt de actie nu niet oneindig door.